### PR TITLE
fix : nav links

### PR DESCRIPTION
**Pull request by ravikumar1002**

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
good to see you are focusing on accessibility  , good to merge !

File: navbar/navbar.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
     position: sticky;
     top: 0;
     left: 0;
-    z-index: 2;
+    z-index: 4;
```
Comment (romabulani) [inline navbar/navbar.css:7]: Any particular reason to give z-index as 4 as the value of z-index doesn't matter as long as there is only on element.

**Review by romabulani — APPROVED** [review]
Looks good to merge

File: navbar/navbar.css (hunk 1 above)
Comment (ravikumar1002) [reply navbar/navbar.css:7]: because I already used 2 somewhere in the project and I want nav always on top so I used

**Pull request merged**